python/mxnet/log.py

```diff
@@ -63,12 +63,38 @@ def format(self, record):
 def getLogger(name=None, filename=None, filemode=None, level=WARNING):
     """Get customized logger.
 
-    Args:
-        name: Name of the logger.
-        level: Level to log.
+    Parameters
+    ----------
+    name: str, optional
+        Name of the logger
+    filename: str, optional
+        The filename to which logging output will be sent
+    filemode: str, optional
+        The file mode to open the file (corresponding to `filename`),
+        default is 'a' if `filename` is not ``None``
+    level: int, optional
+        The `logging` level for the logger
+        See: https://docs.python.org/2/library/logging.html#logging-levels
 
-    Returns:
-        A logger.
+    Returns
+    -------
+    `Logger` object
```

Comment:
This line could just be written as:
Logger
(Backticks are needed only for types that are referenced in paragraphs.)